src/components/PersonLink.tsx

```diff
@@ -0,0 +1,76 @@
+import ClassNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { FC } from 'react';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  findWomen: Person | undefined,
+  findFather: Person | undefined,
```

Comment:
Props 'findWomen' and 'findFather' are not intuitive. Consider renaming these to something more descriptive like 'motherDetails' or 'fatherDetails'.